mkdocs_git_authors_plugin/plugin.py

```diff
@@ -114,17 +104,18 @@ def on_files(self, files, config, **kwargs):
             return
 
         for file in files:
-
             # Exclude pages specified in config
-            excluded_pages = self.config.get("exclude", [])
+            excluded_pages = self.config.exclude or []
             if exclude(file.src_path, excluded_pages):
                 continue
 
             path = file.abs_src_path
             if path.endswith(".md"):
                 _ = self.repo().page(path)
 
-    def on_page_content(self, html, page, config, files, **kwargs):
+    def on_page_content(
+        self, html: str, /, *, page: Page, config: MkDocsConfig, files: Files
```

Comment:
`/` and `*` are cool, but what is `mkdocs` changes the order upstream? At least leave out `/`